docs/server.md

```diff
@@ -0,0 +1,354 @@
+# Neo4j Installation
```

Comment:
If we're going to have this set-up and running instructions seperate from the ReadMe, we should at least link to this doc from the README.